spec/fixtures/tmx/tmx_fixture.rb

```diff
@@ -16,7 +16,9 @@ def render
     end
 
     def wrap_placeholders(text)
-      text.gsub(/(\{\d+\})/) { "<ph>#{$1}</ph>" }
+      text.gsub(/(\{\d+\})/) do
+        "<ph type=\"x-smartling-placeholder\">#{$1}</ph>"
+      end
```

Comment:
I updated this so a test that specifically checks `{0}`-style placeholders would pass with our changes. I don't think it has anything to do with the HTML placeholders.